jdi-light-material-ui-tests/src/test/java/io/github/epam/material/tests/inputs/ButtonTests.java

```diff
@@ -28,137 +35,75 @@ public void before() {
     }
 
     @Test
-    public void defaultButtonTest() {
-
-        containedButtons.get(1).click();
-        containedButtons.get(1).is().text("DEFAULT");
-        containedButtons.get(6).is().text("Last click: Default");
-
-        containedButtons.get(2).click();
-        containedButtons.get(2).is().text("PRIMARY");
-        containedButtons.get(6).is().text("Last click: Primary");
-
-        containedButtons.get(3).click();
-        containedButtons.get(3).is().text("SECONDARY");
-        containedButtons.get(6).is().text("Last click: Secondary");
-
-        containedButtons.get(4).is().text("DISABLED");
-        containedButtons.get(4).is().attr("disabled");
-        containedButtons.get(4).is().disabled();
-
-        containedButtons.get(5).click();
-        containedButtons.get(5).is().text("LINK");
-        containedButtons.get(6).is().text("Last click: Link");
-        containedButtons.get(5).is().notVisible();
+    public void enableContainedButtonTest() {
+        defaultContainedButton.label().is().text("DEFAULT");
+        defaultContainedButton.click();
+        containedButtonLastClick.is().text("Last click: Default");
     }
 
     @Test
-    public void textButtonTest() {
-
-        textButtons.get(1).click();
-        textButtons.get(1).is().text("DEFAULT");
-        textButtons.get(6).is().text("Last click: Default");
-
-        textButtons.get(2).click();
-        textButtons.get(2).is().text("PRIMARY");
-        textButtons.get(6).is().text("Last click: Primary");
-
-        textButtons.get(3).click();
-        textButtons.get(3).is().text("SECONDARY");
-        textButtons.get(6).is().text("Last click: Secondary");
-
-        textButtons.get(4).is().text("DISABLED");
-        textButtons.get(4).is().attr("disabled");
-        textButtons.get(4).is().disabled();
-
-        textButtons.get(5).click();
-        textButtons.get(5).is().text("LINK");
-        textButtons.get(6).is().text("Last click: Link");
-        textButtons.get(5).is().notVisible();
+    public void disableContainedButtonTest() {
+        disabledContainedButton.label().is().text("DISABLED");
+        disabledContainedButton.is().disabled();
+        containedButtonLastClick.is().text("Last click:");
```

Comment:
If the order of the tests is changed (disabled after enabled), then the disabled tests will no longer be passed, since the text field will not only contain "Last click:". There should be checks that will run independently of other tests. Or there will be an explicit order of tests (parameter - priority)

Reply:
As we have @BeforeMethod the page is opened for each test. so the state of the 'Last click' field would be initial for all of the tests